packages/parcel-resolver/src/resolver.tsx

```diff
@@ -0,0 +1,23 @@
+import { dirname, join } from 'path';
+
+import { Resolver } from '@parcel/plugin';
+
+const resolver = new Resolver({
+  async resolve({ dependency }) {
+    const { specifier, resolveFrom } = dependency;
+
+    const cssPrefix = 'compiled-css!';
```

Comment:
How come this prefix has an exclamation mark?

Reply:
actually nvm I read further down and can see here https://github.com/atlassian-labs/compiled/pull/1052/files#diff-c0b739275fb1afe7a00a93f95e32cf560fb7579d0d3a8e7360b8cf003a6e0f24R120 that the dependency is defined with an exclamation mark